docs/README.md

```diff
@@ -0,0 +1,28 @@
+# `docs/`
+
+This folder contains docs for Rill, generated using [Docusaurus](https://docusaurus.io/) and deployed to [https://rilldata.github.io/rill-monorepo-prototype/](https://rilldata.github.io/rill-monorepo-prototype/).
+
+## Local development
+
+To start the docs server with hot reloading, run the following command from the _root_ of the repo:
+
+```
+npm run dev -w docs
+```
+
+## Preview production build
+
+To run a full build and production preview of the docs, run the following commands from the _root_ of the repo:
+
+```
+npm run build -w docs
+npm run preview -w docs
+```
```

Comment:
These commands fail for me as no workspace as been defined in the parent `package.json` file.

Adding 
```
  "workspaces": [
    "docs"
  ]
```

to root `package.json` [after here](https://github.com/rilldata/rill-developer/blob/7d98830d16f8ca3498bdc621d3c22ec1f6a2a0f7/package.json#L127) should fix the issue